src/Sulu/Bundle/MediaBundle/Resources/config/doctrine/FileVersion.orm.xml

```diff
@@ -21,6 +21,7 @@
         </field>
         <field name="storageOptions" type="text" column="storageOptions" nullable="true" />
         <field name="mimeType" type="string" column="mimeType" length="255" nullable="true" />
+        <field name="properties" type="string" column="properties" length="1000" nullable="true" />
```

Comment:
1. You've only added the properties here, shouldn't they be removed from the Media Entity xml configuration as well?
2. Has this change be coordinated with @alexander-schranz ?
3. Why is this necessary?